lib/stages/CMakeLists.txt

```diff
@@ -62,7 +62,6 @@ add_library(
     rfiBroadcast.cpp
     rfiRecord.cpp
     rfiBadInputFinder.cpp
```

Comment:
We could get rid of these 3 stages as well.   They might be useful to refer to in the CHIME upgrade if we want to go this route, but they are based on the data output of the HSA kernels, so at the moment don't do anything.